app/src/lib/stores/app-store.ts

```diff
@@ -160,6 +160,7 @@ import { findAccountForRemoteURL } from '../find-account'
 import { inferLastPushForRepository } from '../infer-last-push-for-repository'
 import { MergeResultKind } from '../../models/merge'
 import { promiseWithMinimumTimeout } from '../promise'
+import { getDefaultRemote } from '../../lib/git/remote'
```

Comment:
This seems pretty close to done - just an unnecessary `import` blocking the compile step